docs/content/tardigrade.md

```diff
@@ -0,0 +1,303 @@
+
+Choose an authentication method.
+Enter a string value. Press Enter for the default ("existing").
+Choose a number from below, or type in your own value
+ 1 / Use an existing access grant.
+   \ "existing"
+ 2 / Create a new access grant from satellite address, API key, and passphrase.
+   \ "new"
+provider> existing
+Access Grant.
+Enter a string value. Press Enter for the default ("").
+access_grant> your-access-grant-received-by-someone-else
+Remote config
+--------------------
+[remote]
+type = tardigrade
+access_grant = your-access-grant-received-by-someone-else
+--------------------
+y) Yes this is OK (default)
+e) Edit this remote
+d) Delete this remote
+y/e/d> y
+```
+
+### Setup with API key and passhprase
+
+```
+No remotes found - make a new one
+n) New remote
+s) Set configuration password
+q) Quit config
+n/s/q> n
+name> remote
+Type of storage to configure.
+Enter a string value. Press Enter for the default ("").
+Choose a number from below, or type in your own value
+[snip]
+XX / Tardigrade Decentralized Cloud Storage
+   \ "tardigrade"
+[snip]
+Storage> tardigrade
+** See help for tardigrade backend at: https://rclone.org/tardigrade/ **
+
+Choose an authentication method.
+Enter a string value. Press Enter for the default ("existing").
+Choose a number from below, or type in your own value
+ 1 / Use an existing access grant.
+   \ "existing"
+ 2 / Create a new access grant from satellite address, API key, and passphrase.
+   \ "new"
+provider> new
+Satellite Address. Custom satellite address should match the format: <nodeid>@<address>:<port>.
+Enter a string value. Press Enter for the default ("us-central-1.tardigrade.io").
+Choose a number from below, or type in your own value
+ 1 / US Central 1
+   \ "us-central-1.tardigrade.io"
+ 2 / Europe West 1
+   \ "europe-west-1.tardigrade.io"
+ 3 / Asia East 1
+   \ "asia-east-1.tardigrade.io"
+satellite_address> 1
+API Key.
+Enter a string value. Press Enter for the default ("").
+api_key> your-api-key-for-your-tardigrade-project
+Encryption Passphrase. To access existing objects enter passphrase used for uploading.
+Enter a string value. Press Enter for the default ("").
+passphrase> your-human-readable-encryption-passphrase
+Remote config
+--------------------
+[remote]
+type = tardigrade
+satellite_address = 12EayRS2V1kEsWESU9QMRseFhdxYxKicsiFmxrsLZHeLUtdps3S@us-central-1.tardigrade.io:7777
+api_key = your-api-key-for-your-tardigrade-project
+passphrase = your-human-readable-encryption-passphrase
+access_grant = the-access-grant-generated-from-the-api-key-and-passphrase
+--------------------
+y) Yes this is OK (default)
+e) Edit this remote
+d) Delete this remote
+y/e/d> y
+```
+
+## Usage
+
+Paths are specified as `remote:bucket` (or `remote:` for the `lsf`
+command.)  You may put subdirectories in too, eg `remote:bucket/path/to/dir`.
+
+Once configured you can then use `rclone` like this.
+
+### Create a new bucket
```

Comment:
These bits of the docs are a nice quickstart for rclone rather than tardigrade specific...

I wonder whether we should move them into a quickstart page? Or include them in all the backends as a quickstart section?

In an ideal world they'd have links to the full docs...

Reply:
I wasn't sure how detailed I should be here. We have this info on our Tardigrade doc site: https://documentation.tardigrade.io/how-tos/sync-files-with-rclone

Should I just replace them with the above link?

Reply:
If you leave a few basic commands like `ls` and `lsf` and link to the docs that sounds like a good idea. Your docs look very nice :-)